### PR TITLE
Expand README with more accurate guides

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,91 @@
...
-When the cluster is available, get the guest kubeconfig using:
+Eventually the cluster's kubeconfig will become available and can be fetched and decoded locally:
 
-```bash
-$ oc get secret --namespace example admin-kubeconfig --template={{.data.value}} | base64 -D
+```shell
+oc get secret \
+  --template={{.data.kubeconfig}} \
+  --namespace clusters \
+  example-admin-kubeconfig | base64 -D
```
Comment (sjenning) [inline README.md:73]: this is a lower `d` on the flag to `base64`

Comment (ironcladlou) [reply README.md:73]: It's -D on BSD (e.g. darwin), -d on GNU, maybe I should just omit that part as a platform-ism?

Comment (ironcladlou) [reply README.md:73]: I stand corrected, on the version of BSD `base64` that ships with 10.15.7 there is support for `-d` and `--decode` as aliases, so how about `--decode`?

Comment (ironcladlou) [reply README.md]: Updated to use `--decode` which hopefully is portable enough

**Pull request merged**